Provide reference link for parameters in config file on node-specific-configurations page (#2407)
risingwavelabs/risingwave-docs: docs/manage/node-specific-configurations.md

  <link rel="canonical" href="https://docs.risingwave.com/docs/current/node-specific-configurations/" />
</head>

> In RisingWave, certain configurations are node-specific and can vary between different nodes. These configurations are stored in a [TOML file](https://github.com/risingwavelabs/risingwave/tree/0be5d74f26526bcaa0b6d18a0cedd627d23cedf7/src/config), which is read at system startup.

Reviewer: Shall we link to the `example.toml` here?

Any items present in `risingwave.toml` will override the default values in the source code. If no configuration file is specified, the default values in `/risingwave/src/common/src/config.rs` will be used.

> To know more details about the parameters in the configuration file, you can visit the [RisingWave configuration files directory](https://github.com/risingwavelabs/risingwave/tree/main/src/config). You will find various information there, like the [definitions and default values of these parameters](https://github.com/risingwavelabs/risingwave/blob/main/src/config/docs.md).

Reviewer 2: ```suggestion
For more details about the parameters in the configuration file, see [RisingWave configuration files directory](https://github.com/risingwavelabs/risingwave/tree/main/src/config). There you'll find information like the [definitions and default values of these parameters](https://github.com/risingwavelabs/risingwave/blob/main/src/config/docs.md).
```
